pydis_site/templates/events/index.html

```diff
@@ -9,58 +9,34 @@
 {% block event_content %}
     <div class="box">
         <h2 class="title is-4">Code Jams</h2>
-        <div class="notification is-success">
-            The 2021 Summer Code Jam qualifier will open June 21st. Check out the details <a href="{% url "events:page" path="code-jams/8" %}">here</a>.
-        </div>
-        <p>Each year, we organize a Winter Code Jam and a Summer Code Jam. During these events, members of our community will work together in teams to create something amazing using a technology we picked for them. One such technology that was picked for the Winter Code Jam 2020 was Kivy, a cross-platform GUI framework.</p>
-        <p>To help fuel the creative process, we provide a specific theme, like <strong>Ancient Technology</strong> or <strong>This App Hates You</strong>. At the end of the Code Jam, the projects are judged by Python Discord server staff members and guest judges from the larger Python community. The judges will consider creativity, code quality, teamwork, and adherence to the theme.</p>
+        <p>Each year, we organize at least one code jam, one during the summer and sometimes one during the winter. During these events, members of our community will work together in teams to create something amazing using a technology we picked for them. One such technology that was picked for the Summer 2021 Code Jam was text user interfaces (TUIS), where teams could pick from a pre-approved list of frameworks.</p>
+        <p>To help fuel the creative process, we provide a specific theme, like <strong>Think Outside the Box</strong> or <strong>Early Internet</strong>. At the end of the Code Jam, the projects are judged by Python Discord server staff members and guest judges from the larger Python community. The judges will consider creativity, code quality, teamwork, and adherence to the theme.</p>
```

Comment:
Why didn't these get cut off like the PyWeek ones?

Reply:
It's not cut off, it just doesn't have an icon to show, so the text takes the full width.
But this did make me realize the Advent of Code icon wasn't showing up, so I'm now fixed that.

Reply:
No I mean that PyWeek's lines you cut off at a certain length, while these ones go on for several hundreds of characters.

Reply:
I'm not sure what you're referencing. The length of the line is entirely dependent on if the word can fit or if it needs to go on the next line. Do you have a screenshot?

Reply:
Why doesn't these lines look like this?
![image](https://user-images.githubusercontent.com/38372706/133911944-6e696f5d-0f82-42d4-8360-7f96512223d5.png)

Reply:
Oh, you mean within the file itself, not how it displays. 

That's just what it was from the start. I only edited the contents of that page and the general order, not how the lines were broken up. I also tend to have word wrap turned on in my editor so it doesn't really bother me.